### DISCUSSION TITLE
Discussion on stock selection algorithms

### DISCUSSION THREAD

**Discussion by akeboshi1** [Stock-Fund/XCrawler#5] [Q&A] [answered]
As the title suggests, you can propose a stock selection algorithm here, preferably one with certain practical value.

**Comment by akeboshi1** [comment]
1. 选择月线/周线为红色的股票为宜，当月线/周线存在上涨放量或者下跌缩量时，说明当前股价可能会产生一波向上的趋势
2. 日线上穿做上影线，试盘尝试突破，后市看涨。下穿做下影线，后市看跌
3. 当两个或者更多大阳线连续出现并形成一个三角形时，如果股票后市趋势向上，股价会形成震荡行情，等待5日线的回归，否则股票的上涨趋势过快会导致股价的快速回落
4. 斐波那契 神奇数字：波那契数列指的是这样一个数列:1、1、2、3、5、8、13、21……这系列数字被称为神奇数字.这个数列从第三项开始，每一项都等于前两项之和、例如1+1=2，1+2=3，2+3=5，3+5=8，5+8=13，8+13=21，13+21=34。利用斐波那契数列时间窗口来推算股票变盘点，也就是通常在指数上，不适用个股上
5. 波那契数列跟黄金分割线结合，形成共振那么可靠性和准确性也是非常高的.无论交易的天数随着时间的推移越来越多还是个股交易的价格涨跌，所有涉及数字的部分都与斐波那契数列和黄金分割有密切的关系。比如，从一个重要的起点开始(可以是顶点也可以是底点)，价格运行到这个价位的重要黄金分割位时都要注意趋势的变化
6. 内盘>外盘 股价放量下跌 赶紧离场
内盘>外盘 股价不跌反涨 庄家在吸筹码
外盘>内盘 股票量价齐涨 后市看涨
外盘>内盘 股价不涨反跌 庄家在出货
内外盘相差不大 股价即将出现反转
内外盘都比较小 股价小幅上涨 代表庄家锁筹随时准备拉升
需要结合一起进行分析，而不是单独作为评判标准

**Comment by happyyf1** [comment, marked as answer]
1. 一般20日均线作为中期生命线，一旦跌破，中期趋势则结束，股价将会有一波调整；但如果20日均线上穿金叉60日均线，表示中期趋势发生反转，后续可能会持续上涨，可以进行建仓/加仓动作
2. 当一只股票均线呈现多头趋势时，表示该股在强势上涨阶段，可以介入，但不易深度介入。
3. 当一只股票在股价底部长时间横盘调整，且均线粘合，表示可能存在的主力在低位吸筹，可等待一个放量上涨的信号介入
4. 当股票的成交量在低位出现缩量的情况时，说明主力控盘较重，后续可能有拉升动作
以上策略需要结合一起进行分析，而不是单独作为评判标准

**Reply by akeboshi1** [reply]
这是个不错的中期择股策略
